datacube_stats/output_drivers.py

```diff
@@ -44,7 +44,7 @@ def polygon_from_sources_extents(sources, geobox):
     # TODO: remove ._geom check once datacube-core is fixed older versions of
     #       datacube returned unusable Geometry object instead of None on
     #       failure
-    if sources_union is None or sources_union._geom is None:  # pylint: disable=protected-access
+    if sources_union is None or sources_union.geom is None:  # pylint: disable=protected-access
```

Comment:
with this change comment above doesn't make sense anymore, I think you need to remove the comment and the part starting from `or ...` including pylint comment

Reply:
Done